python/ray/serve/task_processor.py

```diff
@@ -357,12 +398,38 @@ def initialize(self, config: TaskProcessorConfig):
             reject_on_worker_lost=True,
         )
 
+        queue_config = {
+            config.queue_name: _get_queue_config(config.queue_name),
```

Comment:
nit: would prefer we don't use `_get_queue_config` since it's an unnecessary extra layer that readers have to go through, explicitly creating the dict here is much easier to read